pycbc/detector.py

```diff
@@ -322,3 +304,96 @@ def overhead_antenna_pattern(right_ascension, declination, polarization):
 def effective_distance(distance, inclination, f_plus, f_cross):
     return distance / np.sqrt( ( 1 + np.cos( inclination )**2 )**2 / 4 * f_plus**2 + np.cos( inclination )**2 * f_cross**2 )
 
+    """ LISA class  """
+
+class LISA(object):
+    def __init__(self, kappa, _lambda_, reference_time = 1126259462.0):
```

Comment:
You don't need a reference time here. the purpose was a bit different for the ground based detector.